src/main/java/com/conveyal/gtfs/loader/JdbcTableWriter.java

```diff
@@ -1581,6 +1613,10 @@ private void updateReferencingTables(
                                             referencingTable.name,
                                             entityClass.getSimpleName()
                                     );
+                                    if (patternAndRouteIds.size() > 0) {
+                                        String patternIdsString = StringUtils.join(patternAndRouteIds, ",");
+                                        message += "\n" + "Referenced pattern IDs: [" + patternIdsString + "]";
```

Comment:
Use `String.format`.

Reply:
Done in https://github.com/conveyal/gtfs-lib/pull/378/commits/30754ba741ee414fc1778837160d91e12694eddf